### DISCUSSION TITLE
Erro de Hash na validação do SAF-T

### DISCUSSION THREAD

**Discussion by D-Baptista** [assoft-portugal/SAF-T-AO#245] [General]
Saudações a todos.
Estou testando os arquivos que o meu sistema emite no portal da AGT (para submissão do modelo 8), mas sempre recebo a mensagem de que os hash a partir da segunda fatura estão errados. Já tentei de tudo e nenhuma solução, vi aqui nas discussões e issues algumas soluções, mas não consigo aplicar no meu sistema.
Meu sistema é feito em Java Desktop, alguém que tenha o conhecimento sobre o assunto que possa me ajudar?
Para já agradeço.

**Comment by AfonsoMenata** [comment]
Já passei por isso.

O meu problema foi que usei uma chave privada que não era par da chave publica que está com a agt.

**Reply by D-Baptista** [reply]
Isso já foi ultrapassado, agora a luta é mesmo para adquirir a tal licença, desde que submeti o pedido não tive nenhuma resposta.
